deploy/snapshot/internal/pagebroker/wire_test.go

```
@@ -0,0 +1,112 @@
+		}
+	})
+	if out, err := exec.Command("make", "-C", root, "wire-fixture").CombinedOutput(); err != nil {
+		t.Fatal(string(out))
+	}
+	out, err := exec.Command(filepath.Join(root, "wire-fixture")).Output()
+	if err != nil {
+		t.Fatal(err)
+	}
+	var got Response
+	if err := proto.Unmarshal(out, &got); err != nil || !got.GetOk() || got.GetTransactionId() != "txn" || got.GetDirectoryPath() != "/checkpoint" {
+		t.Fatalf("C++ response: %#v %v", got, err)
```

golangci-lint (2.12.2)

[error] copylocks: call of t.Fatalf copies lock value: github.com/ai-dynamo/dynamo/deploy/snapshot/internal/pagebroker.Response contains google.golang.org/protobuf/runtime/protoimpl.MessageState contains sync.Mutex

(govet)